travel-groups/swagger-ui-router.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var serveStatic = require('serve-static');
+
+// var SWAGGER_DOCS_PATH  = '/:id/swagger-docs/';
+var SWAGGER_UI_PATH    = '/:id/docs/';
```

Comment:
` '/:id/docs/'` route is probably not your route. At which path you want to serve the ui? Learn about Express routing system at their website and fix it.